src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -135,6 +135,11 @@ class ElasticIsotropicUpdates : public SolidBaseUpdates
                                  localIndex const q,
                                  real64 ( &elasticStrain )[6] ) const override final;
 
+  GEOS_HOST_DEVICE
+  virtual void getElasticStrainInc( localIndex const k,
```

Comment:
```suggestion
  virtual void calculateElasticStrainInc( localIndex const k,
```

Since this isn't really a "getter"